components/brave_ads/core/internal/geographical/subdivision/subdivision_feature.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_GEOGRAPHICAL_SUBDIVISION_SUBDIVISION_FEATURE_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_GEOGRAPHICAL_SUBDIVISION_SUBDIVISION_FEATURE_H_
+
+#include "base/feature_list.h"
+#include "base/metrics/field_trial_params.h"
+#include "base/time/time.h"
+
+namespace brave_ads {
+
+BASE_DECLARE_FEATURE(kSubdivisionFeature);
+
+constexpr base::FeatureParam<base::TimeDelta> kSubdivisionFetchIntervalMinutes{
+    &kSubdivisionFeature, "fetch_interval_minutes", base::Days(1)};
```

Comment:
Maybe as a follow up, we fetch on browser startup, and listen for network changes, meaning the value always matches the users Geo.